Draft new survey signal post for review

capnrefsmmat opened the pull request:
To leave comments, go to the Files Changed tab above. You can click and select in the margin (by the line numbers) to choose lines you'd like to comment on:

![Screen Shot 2020-10-05 at 1 35 19 PM](https://user-images.githubusercontent.com/711629/95113000-bb1bd780-070f-11eb-9eca-bb757478260d.png)

You can press a button to make suggestions for specific wording, which I can then go through and select to add:

![Screen Shot 2020-10-05 at 1 35 37 PM](https://user-images.githubusercontent.com/711629/95113040-cc64e400-070f-11eb-82de-bcb67cf98aba.png)

You can press the button to add your suggestions to a "review". When you're done, you can press the green button up top to give a summary and share your comments:

![Screen Shot 2020-10-05 at 1 37 46 PM](https://user-images.githubusercontent.com/711629/95113173-08984480-0710-11eb-936c-d2a350a30eef.png)

GitHub lets us then leave comments on your suggestions and hold discussions in the comments, so we can iterate without changing the Markdown.

capnrefsmmat committed:
Draft new survey signal post for review

capnrefsmmat committed:
Use production signal names

capnrefsmmat committed:
Minor copyedits

capnrefsmmat committed:
Slightly better word choice

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
```diff
@@ -0,0 +1,264 @@
+tags: ["symptom surveys"]
+---
+
+Beginning on September 8, 2020, we deployed a new version of our symptom survey.
```
Before Sept 8, we still have around 35 questions on the survey, and we were still only creating aggregates of data from the "first page" of the survey, related to CLI.  What of the rest of the survey ... is that not being reflected at all in the new aggregates?  (Or have these questions been revised and so they're been reflected in some way, but just not in their original form?)

capnrefsmmat replied:
See https://github.com/cmu-delphi/covidcast-indicators/issues/18 for discussion of which aggregates to release.

I do have other aggregates, on topics like whether the respondent has worked outside their home in the past 5 days, but it was not yet clear to me whether we can release those aggregates. So I defaulted to releasing only the ones that directly reflect COVID activity.

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
```diff
@@ -0,0 +1,264 @@
+Facebook helps us recruit tens of thousands of respondents daily, and the new
+survey gives us unprecedented insights into the effects of COVID-19 across the
+United States. Today we release new public datasets and share maps revealing
+access to COVID testing, changes in mental health, and public use of masks.
```
Would remove reference to mental health here (and throughout) because we're not showing any of this data yet.

capnrefsmmat replied:
Will fix

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
```diff
@@ -0,0 +1,264 @@
+Beginning in early April 2020, the [Delphi group](https://delphi.cmu.edu/) has
+conducted a major survey to track COVID-19 across the United States. With the
+support of Facebook, we have been able to recruit tens of thousands of active
+Facebook users to take our voluntary survey *every day*; meanwhile, a University
```
Small, but unimportant, I would de-italicize every day (it doesn't read that cleanly to me---if you really wanted to emphasize it, I would say "tens of thousands of active Facebook users *every day* to ...").

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
```diff
@@ -0,0 +1,264 @@
+
+In previous posts, we described how [self-reported symptoms on our survey can
+correlate strongly with COVID
+cases](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/)
```
Consider linking to a specific section: jump right to the correlation plots?

capnrefsmmat replied:
I ended up putting a section link later on, when we rely on this specific fact for a plot. I'll keep this link to the whole post, so people can see the context in that post

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
```diff
@@ -0,0 +1,264 @@
+correlate strongly with COVID
+cases](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/)
+and how [symptom survey data could improve forecasts of COVID
+cases](https://delphi.cmu.edu/blog/2020/09/21/can-symptoms-surveys-improve-covid-19-forecasts/).
```
Consider linking to a specific section: jump right to the "error by ahead" plot at the end?

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
```diff
@@ -0,0 +1,264 @@
+  attending public events, working indoors with others, or going to school in
+  person?
+
+Some of these topics were partially addressed by our existing survey, but we
```
Ah ok, so this is a helpful line (which partially addresses my question above, as to what we had before Sept 8), but I still wonder if we could reposition this slightly or if we want to.  Might be more efficient to quickly discuss on a call in OHs, but just flagging it here to remind us.

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
```diff
@@ -0,0 +1,264 @@
+
+Some of these topics were partially addressed by our existing survey, but we
+knew that as the pandemic changes and public health priorities adapt, our survey
+must change also. We began a consultation process: In discussions with
```
```suggestion
must change also. We began a consultation process: in discussions with
```

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
````diff
@@ -0,0 +1,264 @@
+site](https://cmu-delphi.github.io/delphi-epidata/symptom-survey/coding.html)---this
+version is Wave 4.
+
+```{r sample-size, message=FALSE, include=FALSE}
````
Small nitpick, but if this code isn't ever included in the actual blog post, then people won't be able to run the examples if they just copy-paste what they see (since things like `start_day`, `end_day` will be undefined).

capnrefsmmat replied:
I've updated the code so the parts shown in the post should be self-sufficient

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
```diff
@@ -0,0 +1,264 @@
+occupation, and education, allowing us to understand how different groups have
+been affected and which groups are currently most vulnerable to COVID-19. (As we
+[described
+before](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/),
```
Consider linking to a specific subsection: jump right to the part describing this?

capnrefsmmat replied:
It's mentioned in the first paragraph, and then the subsequent section, so I think it's close enough to the top that this link will do

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
````diff
@@ -0,0 +1,264 @@
+```
+
+Because the survey runs continuously, collecting data on mask-wearing every day,
+we will be able to track how the prevalence of mask-wearing changes over time.
````
Consider mentioning or linking to the NY Times article about this?  Suggesting we'll be a able to generate a dynamic version of maps like this?

capnrefsmmat replied:
Added a paragraph to this effect

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
```diff
@@ -0,0 +1,264 @@
+our own [COVIDcast map](https://covidcast.cmu.edu/) also includes data from
+antigen tests produced by Quidel, a major test product manufacturer. But a
+survey allows us to ask people directly whether they have been tested, and
+allows us to understand *who* has been tested.
```
Can you elaborate on this---just show I understand, we're not actually developing on this at all in the examples we're showing right?

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
```diff
@@ -0,0 +1,264 @@
+we will be able to track how the prevalence of mask-wearing changes over time.
+This may help public health officials target public awareness campaigns, and
+also aid researchers trying to understand where the virus is most likely to
+spread and what measures are most effective in preventing it.
```
You had a cool plot at one point that compares mask wearing to CLI or CLI-in-community, with a visually clear inverse relationship.  Consider putting these side-by-side (so as not to exacerbate vertical plot exhaustion)?

capnrefsmmat replied:
Done

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
```diff
@@ -0,0 +1,264 @@
+Dakota's publicly reported testing data says were conducted during that time
+period. But North Dakota's published testing figures [only include PCR tests,
+not COVID antigen
+tests](https://khn.org/news/lack-of-antigen-test-reporting-leaves-country-blind-to-the-pandemic/),
```
Consider linking to article by our friends at the Atlantic: https://www.theatlantic.com/health/archive/2020/09/how-many-people-america-testing-coronavirus/616249/.

capnrefsmmat replied:
Since the KHN article mentions North Dakota specifically, what I'll do instead here is rework the paragraph above a bit. Where I previously mentioned understanding *who* has been tested, which as you point out we don't elaborate on at all, I'll instead link to the Atlantic article and give some motivation using that.

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
````diff
@@ -0,0 +1,264 @@
+```
+
+We can see quite a large variation in test positivity across the United States,
+suggesting different access to testing.
````
Why is a large variation in test positivity suggesting different access?  Some of this could be real signal, right?

Also, we can do a side-by-side plot of the Quidel test positivity signal to compare?

capnrefsmmat replied:
I didn't add Quidel since that starts getting more technical (difference in test types, they're only one manufacturer, etc.), but I clarified the language about test positivity

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
```diff
@@ -0,0 +1,264 @@
+
+We can see quite a large variation in test positivity across the United States,
+suggesting different access to testing.
+
```
In the current version of the post, I noticed you added a sentence explaining the gray states, but I'm confused: why are we missing data here on test positivity, when we have data on who got tested for all states?   Are these derived from answers to separate questions on the survey?

capnrefsmmat replied:
Suppose the "were you tested?" signal has sample size N and population proportion p. The "did you test positive?" signal has sample size at most pN.

But the sample sizes on these will increase and fix most of the grey; we made a change in inclusion criteria that hasn't been applied to the weighted signals yet, and when they're updated, the sample sizes will increase by about 20%.

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
```diff
@@ -0,0 +1,264 @@
+data, we can make decisions needed to protect public health and permit safe
+reopening.
+
+**Credits.** *Many items in Wave 4 of our survey are based on work by the team
```
Small nitpick, would change this to "Acknowledgements", which is the term I switched over to using to label these paragraphs.

capnrefsmmat replied:
Done

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
```diff
@@ -0,0 +1,264 @@
+Morris, gave extensive feedback on Wave 4 of the survey and assisted in its
+deployment. Taylor Arnold, Alex Reinhart, and Kathryn Mazaitis developed the
+code needed to process Wave 4.*
+
```
More substantively, can we refer to (even broadly) and thank all the folks involved in survey design, from the very start, all the way through the current wave?  Weren't there folks from Stanford, Yale, etc.

capnrefsmmat replied:
I actually don't know who was involved in the initial survey design; I wasn't involved then, so I mainly know who was involved in Wave 4.

capnrefsmmat replied:
If you have a list, I'd be happy to add it.

ryantibs commented on content/post/2020-10-06-survey-wave-4.Rmd:
```diff
@@ -0,0 +1,264 @@
+COVIDcast API, alongside numerous other data streams, we hope to provide
+researchers, public health officials, and journalists the information they need
+to form a more complete picture of the pandemic.
+
```
High-level comment: I think what you've have here in terms of examples reads naturally, and is interesting.  Nice job.

But can we connect some of the pieces a bit more to suggest clearer interpretations/findings?  Doesn't even need to be all that concrete, just moving in that direction.  The idea of pitting mask wearing vs CLI, test positivity from our survey vs Quidel, etc., were moving in that direction ... I'm sure there's other ways to do this too.  Let me know if you want to discuss.  Maybe some time series plots showing trends over time would also provide a nice "break" from the many maps (and highlight the temporal aspect).

capnrefsmmat replied:
If you have any specific suggestions for plots or findings, I'd be glad to hear them. I just tried a few time series plots, but not much happens over the month we have available. (Frustratingly, all the state time series for masks are completely flat -- hardly any change over the past month.) But if there are two signals that would be interesting to correlate, I can definitely try it out and make a plot.

ryantibs reviewed (commented):
These are suggestions: feel free to push back and/or discuss before embarking on anything nontrivial.

Also one overall comment that I forgot to make in any one specific place: as a subtitle on all the plots, you might consider placing a subtitle that says "Based on X responses" where X is the sample size used for that indicator.  Would make salient the claims about this being a massive survey.

capnrefsmmat committed:
Remove mental health mentions

capnrefsmmat committed:
Small copyedits suggested by Ryan

capnrefsmmat committed:
Change time interval to include roughly 1 full month

capnrefsmmat committed:
Improve mask maps, add extra plot, and expand discussion

capnrefsmmat committed:
Link to Atlantic story on test reporting

capnrefsmmat committed:
Clarify language about test positivity rates

capnrefsmmat committed:
Credit people who suggested Wave 4 revisions

zackchase committed:
Update 2020-10-06-survey-wave-4.Rmd

light edits

capnrefsmmat committed:
Cuter title

capnrefsmmat committed:
Spelling fix

capnrefsmmat committed:
A couple helpful links

capnrefsmmat committed:
Refer to UMD's upcoming blog post

capnrefsmmat committed:
Better credits, date

capnrefsmmat committed:
Merge branch 'main' into survey-2

capnrefsmmat committed:
Show a point for each state

Suggested by Ryan to make the plot clearer.

capnrefsmmat committed:
More clear language about confounding

capnrefsmmat committed:
Switch to plotly interactive plot

Note the need to move the summary into the YAML header. Without this,
plotly puts its JavaScript at the top of the file -- and that JS is put
in the summary and not the post content!

capnrefsmmat committed:
Additional credits

capnrefsmmat committed:
Clarify Facebook's role slightly

capnrefsmmat committed:
Remove unnecessary Plotly buttons

capnrefsmmat committed:
Add line of best fit back to plot

capnrefsmmat committed:
Advertise the COVIDcast API's other signals while we have the chance

capnrefsmmat committed:
Corrected testing data and improved discussion of it

capnrefsmmat committed:
Don't imply generalization from the survey data to population

capnrefsmmat committed:
Suggest clicking each point in the interactive plot; clarify language

Account for biases in the survey.

capnrefsmmat committed:
Merge branch 'main' into survey-2

capnrefsmmat committed:
Rebuild HTML
